### DISCUSSION TITLE
Shade/Roll up windows

### DISCUSSION THREAD

**Discussion by erikjanp** [OpenRCT2/OpenRCT2#16432] [Feature requests]
On Linux, you can shade/roll up windows: the window is rolled up into the title bar, and this title bar stays visible. This makes it easy to take a quick look beneath a window without having to move it, or to have a lot of windows open without them taking up too much space.
Here are two patches that add this functionality to OpenRCT2: the first patch adds the technical aspects of shading windows; the second patch makes most of the windows shadable.
Shading/unshading can be done via a new button on the title bar of the window, by using your mouse scroll wheel on the title bar, or by using the keybindings ';' (shade/unshade active window) or '[' (shade/unshade all windows).
Patches are against v0.3.5.1
https://forums.openrct2.org/applications/core/interface/file/attachment.php?id=8961&key=36a4c2b116e3e3191411e2d110d2fd35
https://forums.openrct2.org/applications/core/interface/file/attachment.php?id=8962&key=05fd118b23eb0038920c7aeea8f1937c

**Comment by duncanspumpkin** [comment]
Ignoring the fact that this is a patch and not a pull request as requested in your other issues, is there actually any appetite for this sort of feature.

**Reply by 733737** [reply]
I never use this in linux but I think in general the UI is lacking in features and it would be nice if you could cascade or snap windows or if you could have rounded corners that are skinnable similar to rct classic

**Reply by ocalhoun6** [reply]
Seems like the kind of thing that would be better suited for a plugin ... if the plugin system is able to accomplish it.

--also linux user who *never* uses the 'shade window' feature.

**Reply by 733737** [reply]
the problem with plugins is that you are limiting your userbase. Someone new to openrct2 could benefit from rounded corners

**Reply by ocalhoun6** [reply]
I mean window shading as a plugin feature. The userbase for that is _already_ very limited.

Rounded corners/fully skinnable windows would make more sense as a feature of the base game ... though to fully implement it seems monstrously difficult to me. (And anyway, if you were to go that route, you'd probably want people to be able to download 'theme' files ... which would basically be a different kind of plugin.)

**Reply by 733737** [reply]
the game already has themes that you can save and theoretically share

**Reply by YJSoft** [reply]
I agree. shading is useful in linux because closing window sometimes stops execution. However, at OpenRCT2, closing any kind of window does not affect gameplay at all.(excepting entering / exiting construction mode but you can always reopen window again)

Also some non-linux users might think shading as "bug" (my window content disappeared! or something)

**Reply by 733737** [reply]
I rather have the option to implement an autoclose timer, I have built entire parks in a 12 hour speedrun, and the information kiosk 1 window was open for 11 hours
